src/main/webapp/index.html

```diff
@@ -16,24 +16,24 @@
 
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
     <link rel="stylesheet" href="/css/main.css">
     <script src="/js/navigation-loader.js"></script>
-  </head>
-  <body onload="addLoginOrLogoutLinkToNavigation();">
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-      </ul>
+</head>
+
+<body onload="addLoginOrLogoutLinkToNavigation();">
+    <nav id="navigation">
+        <ul>
+            <li><a href="/">Home</a></li>
+        </ul>
     </nav>
-    <h1>CodeU Starter Project</h1>
-    <p> Hello World!! This is the CodeU starter project. Click the links above to login and visit your page.
-    You can post messages on your page, and you can visit other user pages if you have
-    their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <p>CodeU Starter Project</p>
+    <p> Hello World!! This is the CodeU starter project. Click the links above to login and visit your page. You can post messages on your page, and you can visit other user pages if you have their URL.</p>
+    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative! You could start by modifying this page to tell the world more about your team.</p>
     <p>Im Editing</p>
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
here (this means add a new line)

Reply:
I dont know it does that automatically when I save the file in visual studio, I can not make the change.